build.gradle

```diff
@@ -1,37 +1,61 @@
 plugins {
     id "io.codearte.nexus-staging" version "0.21.2"
+    id 'eu.xenit.amp' version '1.0.1' apply(false)
+    id 'eu.xenit.alfresco' version '1.0.1' apply(false)
+    id 'eu.xenit.docker-alfresco' version '5.0.6' apply(false)
+    id 'eu.xenit.docker-compose.auto' version '5.0.6' apply(false)
 }
 
 nexusStaging {
     username = project.hasProperty('publish_username') ? project.publish_username : ''
     password = project.hasProperty('publish_password') ? project.publish_password : ''
 }
 
-subprojects {
-    group = 'eu.xenit.alfresco'
-    version = '0.1.0'
-
-    boolean isRelease = System.env.BRANCH_NAME?.startsWith("release")
-    if (!isRelease)
-        version += "-SNAPSHOT"
+ext {
+    shortDotLessAlfrescoVersion = project.hasProperty("alfVersion") ? project.property("alfVersion") : '61'
+    care4alfModulesVersion = '2.1.0'
+    lombokVersion = '1.18.10'
+}
 
+subprojects {
     tasks.withType(JavaCompile) {
         options.encoding = 'UTF-8'
     }
 
     ext {
         junitJupiterVersion = '5.6.2'
         mockitoVersion = '3.3.3'
-        hamcrestVersion = '2.2'
+        hamcrestVersion = '1.3'
         restAssuredVersion = '4.3.0'
     }
 
+    apply from: "${rootProject.projectDir}/publish.gradle"
+}
+
+apply from: "${project.rootProject.projectDir}/integration-tests/alfresco-community-${shortDotLessAlfrescoVersion}/overload.gradle"
+
+allprojects {
+    group = 'eu.xenit.alfresco'
+    version = '0.1.0'
+
+    boolean isRelease = System.env.BRANCH_NAME?.startsWith("release")
+    if (!isRelease)
+        version += "-SNAPSHOT"
+
     repositories {
         mavenCentral()
+        maven {
+            url "${rootProject.projectDir}/alfresco-bom-repo"
+        }
         maven {
             url 'https://artifacts.alfresco.com/nexus/content/groups/public'
         }
+        maven {
```

Comment:
Do we need access to our internal artifactory?
Currently, I only see dependencies on public artifacts.

Reply:
Removed!